cmd/frontend/db/users_test.go

```diff
@@ -98,41 +98,52 @@ func TestUsers_Create_checkPasswordLength(t *testing.T) {
 	dbtesting.SetupGlobalTestDB(t)
 	ctx := context.Background()
 
-	minPasswordRunes := 12
-	cfg := conf.Get()
-	cfg.AuthMinPasswordLength = minPasswordRunes
-	conf.Mock(cfg)
-	defer func() {
-		cfg.AuthMinPasswordLength = 0
-		conf.Mock(cfg)
-	}()
-
+	minPasswordRunes := conf.AuthMinPasswordLength()
 	expErr := fmt.Sprintf("Passwords may not be less than %d or be more than %d characters.", minPasswordRunes, maxPasswordRunes)
 	tests := []struct {
 		name     string
+		username string
 		password string
+		enforce  bool
 		expErr   string
 	}{
 		{
 			name:     "exceeds maximum",
+			username: "user1",
 			password: strings.Repeat("x", maxPasswordRunes+1),
+			enforce:  true,
 			expErr:   expErr,
 		},
 		{
 			name:     "below minimum",
+			username: "user2",
 			password: strings.Repeat("x", minPasswordRunes-1),
+			enforce:  true,
 			expErr:   expErr,
 		},
 
 		{
 			name:     "no problem",
+			username: "user3",
+			password: strings.Repeat("x", minPasswordRunes),
+			enforce:  true,
+			expErr:   "",
+		},
+		{
+			name:     "does not enforce",
+			username: "user4",
 			password: strings.Repeat("x", minPasswordRunes),
```

Comment:
Doesn't this need to be minPasswordRunes-1 to test lack of enforcement? I would add a similar test for the maximum.

```suggestion
			password: strings.Repeat("x", minPasswordRunes-1),
```